pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -1549,6 +1550,22 @@ func Convert_garden_Shoot_To_v1alpha1_Shoot(in *garden.Shoot, out *Shoot, s conv
 	out.Spec.SecretBindingName = in.Spec.Cloud.SecretBindingRef.Name
 	out.Spec.SeedName = in.Spec.Cloud.Seed
 
+	if email, ok := in.Annotations[constants.AnnotationShootOperatedBy]; ok {
+		if in.Spec.Monitoring.Alerting.EmailReceivers != nil {
```

Comment:
I see your defaulting, though, I'd implement `nil` checks here on the various levels to prevent undesired panics in case the defaulting is changed or removed in the future